packages/embark/src/lib/core/plugins.js

```diff
@@ -17,11 +17,20 @@ var Plugins = function(options) {
   this.version = options.version;
 };
 
+Plugins.deprecated = {
+  'embarkjs-connector-web3': '4.1.0'
+};
```

Comment:
As mentioned in the other comment, I'm not actually sure if we want to deprecate this package. As far as I understood, the goal was just to make it **temporarily** not mandatory to list it as an explicit dependency of a DApp. However, in v5 it'll need to be re-added.

Am I missing something here?

Reply:
I'll explain in another reply to your bulleted comment above.